### ISSUE TITLE
Add assay_types.yaml for all the brackets on production

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/search-schema#23]
```
[sc_atac_seq_snare]
[salmon_rnaseq_snareseq]
[seqFISH_lab_processed]
[salmon_rnaseq_bulk]
[salmon_rnaseq_sciseq]
```
(there is also `['ATACseq-bulk]`... but I'm going to assume that's just wrong: Need to ping Bill...)

**Comment by ngehlenborg** [comment]
Additionally, "seqFish" needs to be mapped to "seqFISH".

**Comment by ngehlenborg** [comment]
Here are the mappings that should be used:

```
[sc_atac_seq_snare] = scATAC-seq (SNARE-seq) [SnapATAC]
[salmon_rnaseq_snareseq] = scRNA-seq (SNARE-seq) [Salmon]
[seqFISH_lab_processed] = seqFISH [Lab Processed]
[salmon_rnaseq_bulk] = Bulk RNA-seq [Salmon]
[salmon_rnaseq_sciseq] = scRNA-seq (sciSeq) [Salmon]
```

**Comment by mccalluc** [comment]
Fixed by #25. (search-api will still need to be updated, and then a reindex...)
